YoCode/Output/IPrint.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace YoCode
+{
+    interface IPrint
+    {
+        void AddIntro(string intro);
+        void AddFeature(FeatureData data);
+        void AddMessage(string msg);
+        void AddErrs(IEnumerable<string> err);
```

Comment:
No need to abbreviate to Errs here, better to say the whole word Errors to avoid possible confusion.